yarn.lock

```diff
@@ -2475,12 +2475,12 @@
     pocket-js-core "0.0.3"
     web3-provider-engine "16.0.1"
 
-"@reduxjs/toolkit@^1.5.0":
-  version "1.6.1"
-  resolved "https://registry.yarnpkg.com/@reduxjs/toolkit/-/toolkit-1.6.1.tgz#7bc83b47352a663bf28db01e79d17ba54b98ade9"
-  integrity sha512-pa3nqclCJaZPAyBhruQtiRwtTjottRrVJqziVZcWzI73i6L3miLTtUyWfauwv08HWtiXLx1xGyGt+yLFfW/d0A==
+"@reduxjs/toolkit@^1.6.2":
```

Comment:
**question** was this upgrade necessary?

Reply:
Not necessary, but I try to keep this package up-to-date when I mess around in Redux-land.